core/modules/entity/entity.module

```diff
@@ -282,6 +282,16 @@ function entity_get_info($entity_type = NULL) {
           }
         }
 
+        // Add a token view mode if it does not already exist. Only work with
+        // fieldable entities.
+        if (!empty($entity_info[$name]['fieldable'])) {
+          if (!isset($entity_info[$name]['view modes']['token'])) {
+            $entity_info[$name]['view modes']['token'] = array(
+              'label' => t('Tokens'),
+            );
```

Comment:
Other places, `'custom settings' => FALSE,` is also added. Should that be here too?

Additionally, if the `token` view mode is added to all entities here, do they need to be specified separately in node, comment, user, and taxonomy modules?

Reply:
Done